posts/2024/2024_06_21_Inigo_week_4.rst

```diff
@@ -0,0 +1,48 @@
+Week 4 into GSoC 2024: Weight transfer experiments, hardships, and results!
+===========================================================================
+
+.. post:: June 21 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
```

Comment:
It's just that it's a bit tight, so if you could put spaces, it would be better for reading.